src/boot/libu64/mtxuty-cvt.c

```diff
@@ -0,0 +1,21 @@
+#include "libu64/mtxconv.h"
+
+void MtxConv_F2L(Mtx* mtx, MtxF* mf) {
+    s32 i;
+    s32 j;
+
+    for (i = 0; i < 4; i++) {
+        for (j = 0; j < 4; j++) {
+            s32 value = (mf->mf[i][j] * 0x10000);
+            u16(*intPart)[4] = (u16(*)[4])mtx->m;
+            u16(*fracPart)[4] = (u16(*)[4])mtx->m[2];
```

Comment:
sys_matrix handles this in a cleaner way by casting to a struct. What do you think about it?
```c
    struct {
        s16 intPart[4][4];
        u16 fracPart[4][4];
    }* mu = (void*)mtx;
```

Reply:
Ahh didn't realize we had a similar solution